app/models/User.js

```diff
@@ -1057,6 +1057,27 @@ module.exports = (User = (function () {
       return value
     }
 
+    getEducatorSignupExperimentValue () {
```

Comment:
we should add condition of english speaking and new users like the other experiment: getHomePageExperimentValue